valve-monitor/web-app/src/pages/api/device/[deviceUID]/valve-monitor.ts

```diff
@@ -57,16 +57,17 @@ function validateRequest(
     return false;
   }
 
-  return { deviceUID, valveMonitorConfig, name, valveState };
+  return { deviceUID, valveMonitorConfig, name, state };
 }
 
 async function performPostRequest({
   deviceUID,
   valveMonitorConfig,
   name,
-  valveState,
+  state,
 }: ValidRequest) {
   const app = services().getAppService();
+  console.log("valve monitor file ", state);
```

Comment:
I think you accidentally left this in there.

Reply:
Thanks, removed it again